superset-frontend/src/filters/components/GroupBy/GroupByFilterPlugin.tsx

```diff
@@ -18,13 +18,11 @@
  */
 import { ensureIsArray, ExtraFormData, t, tn } from '@superset-ui/core';
 import React, { useEffect, useState } from 'react';
-import { Select } from 'src/common/components';
 import { FormItemProps } from 'antd/lib/form';
-import { Styles, StyledSelect, StyledFormItem, StatusMessage } from '../common';
+import { Select } from 'src/components';
+import { Styles, StyledFormItem, StatusMessage } from '../common';
```

Comment:
Hmm... this "Styles" could probably use a more specific name.

Reply:
Done